### PR TITLE
added uninstall command support on make

### PR DESCRIPTION
**Pull request by Mte90**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -25,6 +25,13 @@ ifeq ($(UNAME), Linux)
 	$(INSTALLDATA) 19-footswitch.rules /etc/udev/rules.d
 endif
 
+uninstall: all
```
Comment (rgerganov) [inline Makefile]: this shouldn't depend on `all`

File: Makefile [hunk 2]
```diff
@@ -25,6 +25,13 @@ ifeq ($(UNAME), Linux)
...
+	rm /usr/local/bin/footswitch
```
Comment (rgerganov) [inline Makefile]: rm -f ...

**Comment by Mte90** [comment]
fix done :-)

**Comment by Mte90** [comment]
ping

**Pull request merged**